screens/rewards/RewardsScreen.js

```diff
@@ -73,6 +82,42 @@ export default class RewardsScreen extends React.Component {
       return null;
     }
 
+    if (this.state.isGuest) {
+      return (
+        <View style={{ flex: 1 }}>
+          <NavHeaderContainer vertical backgroundColor={Colors.primaryGreen}>
+            <NavButton onPress={() => this.props.navigation.goBack()}>
+              <FontAwesome5 name="arrow-down" solid size={24} color="white" />
+            </NavButton>
+            <BigTitle
+              style={{
+                marginLeft: 18,
+                color: Colors.lightest,
+                fontSize: 36,
+              }}>
+              How it Works
+            </BigTitle>
+          </NavHeaderContainer>
```

Comment:
Update this to match the designs. It should be a regular size `NavHeaderContainer` with a regular `NavTitle` saying 'Healthy Rewards'.